sys/linux/dev_msm.txt.const

```diff
@@ -1,62 +1,62 @@
 # Code generated by syz-sysgen. DO NOT EDIT.
-arches = 386, amd64, arm, arm64, mips64le, ppc64le, riscv64, s390x
+arches = arm64
```

Comment:
Const files need to be generated for all arches. Please run make extract.

Reply:
Fixed